src/main/java/net/liquidpineapple/pang/screens/LevelEditor.java

```diff
@@ -0,0 +1,227 @@
+package net.liquidpineapple.pang.screens;
+
+import net.liquidpineapple.pang.Application;
+import net.liquidpineapple.pang.InputHandler;
+import net.liquidpineapple.pang.XmlHandler;
+import net.liquidpineapple.pang.objects.Ball;
+import net.liquidpineapple.pang.objects.BallMovement;
+import net.liquidpineapple.pang.objects.GameObject;
+import net.liquidpineapple.pang.objects.Player;
+import net.liquidpineapple.pang.objects.playerschemes.Player1;
+
+import java.awt.Graphics2D;
+import java.awt.Image;
+import java.awt.MouseInfo;
+import java.awt.Point;
+import java.awt.PointerInfo;
+import java.awt.image.ImageObserver;
+import java.io.IOException;
+import java.util.LinkedList;
+
+import javax.imageio.ImageIO;
+
+/**
+ * Class that represents the LevelEditor.
+ *
+ * @author Jaap-Jan
+ * @date 10-10-2016.
+ */
+public class LevelEditor extends Screen {
+  public LinkedList<GameObject> addedObjects = new LinkedList<>();
+  private static boolean addedPlayer = false;
+
+  /**
+   * Constructor for the LevelEditor screen.
+   * Tries to set background.
+   */
+  public LevelEditor() {
+    try {
+      backgroundImage = ImageIO.read(LevelScreen.class.getResource("/sprites/editorbg.png"));
+    } catch (IOException ex) {
+      ex.printStackTrace();
+    }
+  }
+
+  /**
+   * Method that updates the Leveleditor.
+   */
+  @Override
+  public void doUpdate() {
+    super.doUpdate();
+    PointerInfo pointerInfo = MouseInfo.getPointerInfo();
+    Point point1 = pointerInfo.getLocation();
+    Point point2 = Application.getBoard().getLocationOnScreen();
+    int intXpos = (int) (point1.getX() - point2.getX());
+    int intYpos = (int) (point1.getY() - point2.getY());
+    if (InputHandler.isAnyKeyPressed()) {
+      BallMovement ballMovement = BallMovement.valueOf("LEFT_MOVEMENT");
+      //key 1 pressed, add ball with size 1
+      if (InputHandler.getKeysDown().contains(49)) {
+        InputHandler.getKeysDown().clear();
+        int size = 1;
+        int offset = 0;
+        try {
+          Image ballImage = ImageIO.read(LevelScreen.class.getResource("/sprites/Balls/green.png"));
+          offset = ballImage.getWidth(null) / 2;
```

Comment:
Instead of: 
`Image ballImage = ImageIO.read(LevelScreen.class.getResource("/sprites/Balls/green.png"));`
You do: 
`Ball ball = New Ball(0,0, ballMovement, size)`
`offset = ball.getWidth()/2`
`Ball.setX(currentmouseX - offset)`
`Ball.setY(currentmouseY - offset)`